mllib/src/test/scala/org/apache/spark/ml/tuning/ParamGridBuilderSuite.scala

```diff
@@ -59,5 +59,9 @@ class ParamGridBuilderSuite extends FunSuite {
       (10, "input1"),
       (20, "input1"))
     validateGrid(maps1, expected1)
+
+    val maps2 = (new ParamGridBuilder()).build()
```

Comment:
Trivial: The initial set of parens isn't needed. I think you can write just `Set.empty[(Int,String)]` alone?